### PR TITLE
List of favorites

### PR DESCRIPTION
**Pull request by rachwyatt**
@stenington

### CODE REVIEW

**Review by stenington — CHANGES_REQUESTED** [review]
Good start! I think you should just go to `localhost:8000/` to see the list of favorites though, and the page it shows should be valid HTML.

**Comment by rachwyatt** [comment]
@stenington Because my html didn't have html, head and body? I get in my head that html should be simple and then realize i have no idea what i'm doing. Still need to get set up with that html validator thing you were talking about.

File: favorites/templates/favorites/favorites.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<html>
```
Comment (stenington) [inline favorites/templates/favorites/favorites.html:2]: The first line of this file should be `<!DOCTYPE html>` for it to be valid, with your `<html>` tag on the second line.

**Review by stenington — CHANGES_REQUESTED** [review]
For HTML5, the smallest valid document is apparently a `<!DOCTYPE html>` tag and then a non-empty `<title> </title>` tag. I think it's a good idea to use `<html>`, `<head>`, and `<body>` tags too though.

**Comment by rachwyatt** [comment]
@stenington I added the `<!DOCTYPE html>`

**Review by stenington — APPROVED** [review]

**Pull request merged**